apps.yml

```diff
@@ -3885,16 +3885,6 @@ apps:
     - team_mofo
     - team_mozillaonline
     authorized_users: []
-    client_id: axRGaXPRd5pe60fnEllluSN76MefFX1E
```

Comment:
If this line drops, the resulting entry has no `client_id`.  Intentional?

Reply:
Thanks for catching that! I had accidentally deleted one more line than I intended.